angular.json

```diff
@@ -117,7 +117,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "10mb",
-                  "maximumError": "23mb"
```

Comment:
We've only bumped mermaid one point release. The size increase has been cumulative. We should investigate soon.